docs/detections/rules-ui-create.asciidoc

```diff
@@ -289,7 +289,9 @@ TIP: Click the help icon (image:images/esql-ref-button.png[Click the ES|QL help
 [[esql-agg-query]]
 ===== Aggregating query 
 
-Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by an {esql} rule with an aggregating query only contain the fields returned by the query.
+Aggregating queries use {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate source event data. Alerts generated by a rule with an aggregating query only contain the source event fields that the {esql} query returns and new fields that the query creates.
+
+NOTE: New fields are fields that don't exist in the query’s source index. For example...
```

Comment:
example can be taken from here: https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-commands.html#esql-stats-by

```
FROM employees
| STATS count = COUNT(emp_no) BY languages
| SORT languages

```

Here is a new column/field `count` is created

Reply:
```suggestion
NOTE: New fields are fields that don't exist in the query’s source index. For example, if you use the {ref}/esql-commands.html#esql-stats-by[`STATS...BY`] function to create a column with an aggregated value, the newly-added column is considered a new field. 
```